demo/basic/generated/swift/IHtmlApi.swift

```diff
@@ -85,6 +85,20 @@ public class IHtmlApi {
     )
     jsExecutor.execute(with: "htmlApi", feature: "testDictionaryWithAnyKey", args: args, completion: completion)
   }
+
+  public func testDefaultValue(bool: Bool? = nil, bool2: Bool? = nil, num: Double = 1, completion: BridgeJSExecutor.Completion? = nil) {
```

Comment:
I don't prefer to assign nil to `bool2` by default. For example, a new added optional config without default value, should caused compile error, so the caller can realize the interface change.

Reply:
I think that if the new config is necessary, then it shouldn't be optional, and if it is optional, it means the new ingestion should work fine without it, and the caller can just modify it when necessary

Reply:
For example, in one recent PR, one reviewer asked me to change a config from `optional bool = false` to `optiaonal bool`, because the component is used in differnt places, if a defualt config is preset, then the we may miss some change

Reply:
As what I said, if this config is necessary for other caller, it should be `bool` instead of `optional bool`

Reply:
But I get your point, caller should handle every config update explicitly, let me remove it.